doc/advanced-topics.md

```diff
@@ -418,6 +418,27 @@ For each attribute in the `reverse` array, an expression must be defined to calc
 notification attributes. This value will be passed to the underlying protocol with the `object_id` name. Details about
 how the value is then progressed to the device are protocol-specific.
 
+#### Autoprovision configuration (autoprovision)
+
+By default, when a measure arrives to the IoTAgent, if the `device_id` does not match with an existing one, then, the IoTA 
+creates a new device and a new entity according to the group config. Defining the parameter `autoprovision` to `false` 
+when provisioning the device or the group, the IoTA to reject the measure at the southbound, allowing only to persist the 
+data to devices that are already provisioned. 
+
```

Comment:
I'd suggest to add the following paragraph:

> `If `autoprovision` is provided both at device and group level, the device level takes precedence

Does it work that way, right?

Reply:
Just reading your review, I just realized It does not have sense to define autoprovision at device leve since this means the device is arlready provisioned.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1068/commits/1c058365f8b3fd607d91def65af548f7c86a27d8

Reply:
> Just reading your review, I just realized It does not have sense to define autoprovision at device leve since this means the device is arlready provisioned.

Good point :)

Maybe it should be even explicitly mentioned.

Reply:
Added in https://github.com/telefonicaid/iotagent-node-lib/pull/1068/commits/9ecb677e15323c82b88a0b1dce2fc620287fa1ec